Packs/AnythingLLM/Integrations/AnythingLLM/AnythingLLM.yml

```diff
@@ -204,7 +216,48 @@ script:
       - 'false'
       description: Set pin status to true or false.
     description: Set the pinned status of a document embedding.
-  dockerimage: demisto/python3:3.11.9.104657
+  - arguments:
+    - description: Name of the workspace.
+      name: workspace
+      required: true
+    - description: Name of the new conversation thread.
+      name: thread
+      required: true
```

Comment:
Generally speaking, if we add a required argument to an existing command, it will break backwards compatibility. If it's necessary we can create a BC break notice by using the -bc flag in the update-release-notes demisto-sdk command.

Reply:
If needed since very new pack with only 15 downloads, can add BC notice.